ariadne_graphql_proxy/contrib/imgix/query_params_resolver.py

```diff
@@ -0,0 +1,59 @@
+from functools import partial
+from typing import Any, Callable, Optional, Union, cast
+from urllib.parse import parse_qs, urlencode, urlparse, urlunparse
+
+from graphql import GraphQLResolveInfo
+
+
+def get_attribute_value(
+    obj: Any, info: GraphQLResolveInfo, attribute_str: str, **kwargs
+) -> Any:
+    value = obj
+    for attr in attribute_str.split("."):
+        value = value.get(attr)
```

Comment:
We should have try/except test and do `getattr(value, attr, None)` in except if `get()` fails. This is to support other types than dicts.

Reply:
Done, I also added tests with non dict objects and mentioned them in `GUIDE.md`